asv_bench/benchmarks/io/csv.py

```diff
@@ -98,6 +99,44 @@ def time_frame_date_no_format_index(self):
         self.data.to_csv(self.fname)
 
 
+class ToCSVPeriod(BaseIO):
+    fname = "__test__.csv"
+
+    params = ([1000, 10000], ["D", "H"])
+    param_names = ["obs", "fq"]
+
+    def setup(self, obs, fq):
+        rng = period_range(start="2000-01-01", periods=obs, freq=fq)
+        self.data = DataFrame(rng)
+
+    def time_frame_period_no_format(self, obs, fq):
+        self.data.to_csv(self.fname)
+
+    def time_frame_period_formatting(self, obs, fq):
+        """IMPORTANT: date_format is not actually taken into account here today
```

Comment:
can you make this a comment instead of a docstring.

not sure what to make of IMPORTANT here.  i often preface comments with "Note:" or "NB:" depending on what i think the reader needs to know.

Reply:
ok. I tried something, let me know if it is better